tools/musictools.py

```diff
@@ -373,6 +381,11 @@ def hsmtxt_to_yaml(args):
             lambda match: f'{match.group(1)}"{escYamlScalar(match.group(2))}"',
             txt, flags=re.MULTILINE
         )
+        txt = re.sub(  # Artist "Yes" wreaks heck
```

Comment:
Rejected in favor of ac92e7d299f502df9e43680d57ffa04337fc0307 (treat "Yes" as a must-escape token)